### PR TITLE
Feature: Add helper scripts

### PR DESCRIPTION
**Pull request by methylDragon**
Changes reflected in the README updates!

:fire: :dragon_face:

### CODE REVIEW

**Comment by methylDragon** [comment]
Sorry, wait up, there are build bugs. Let me resolve them.

**Comment by methylDragon** [comment]
Ok, good to go!

File: blue_helpers/package.xml [hunk 1]
```diff
@@ -1,8 +1,7 @@
-<?xml version="1.0"?>
-<package>
+<package format="2">
   <name>blue_helpers</name>
   <version>0.0.0</version>
-  <description>The blue_helpers metapackage contains helpful tools for running the Blue robot arm</description>
+  <description>The blue_helpers package contains helpful tools for the Blue robot arm</description>
 
   <maintainer email="phil80301@berkeley.edu">Philipp Wu</maintainer>
```
Comment (wuphilipp) [inline blue_helpers/package.xml:6]: Your welcome to change the maintainer and author to yourself @methylDragon. It's your contribution!

**Review by wuphilipp — COMMENTED** [review]
Otherwise looks great!

File: blue_helpers/package.xml (hunk 1 above)
Comment (methylDragon) [reply blue_helpers/package.xml:6]: Oh! Alrighty then, cheers!

**Review by wuphilipp — APPROVED** [review]

**Pull request merged**